website/docs/cloud-docs/run/api.mdx

```diff
@@ -191,6 +191,6 @@ rm ./create_config_version.json
 
 ## Advanced Use Cases
 
-For advanced use cases see the [TFE Automation Script](https://github.com/hashicorp/terraform-guides/tree/master/operations/automation-script) repository for automating interactions with HCP Terraform, including the creation of a workspace, uploading code, setting variables, and triggering a plan/apply.
+For advanced use cases see the [Terraform Enterprise Automation Script](https://github.com/hashicorp/terraform-guides/tree/master/operations/automation-script) repository for automating interactions with HCP Terraform, including the creation of a workspace, uploading code, setting variables, and triggering a plan/apply.
```

Comment:
```suggestion
For advanced use cases refer to the [Terraform Enterprise Automation Script](https://github.com/hashicorp/terraform-guides/tree/master/operations/automation-script) repository for automating interactions with HCP Terraform, including the creation of a workspace, uploading code, setting variables, and triggering the `plan` and `apply` operations.
```
Small thing, but we need to start thinking more about the language we use w/r/t TF operations.